### ISSUE TITLE
Unoptimized asset files

### ISSUE THREAD

**Issue by sahithyandev** [moraexams/website#20]
Tried opening moraexams.org yesterday and it took too long to load. It's because the asset files are not optimized. They add up to a total of 14MB. Especially the favicon shouldn't be more than 8kb.

![Screenshot from 2024-07-18 00-48-10](https://github.com/user-attachments/assets/2c438aeb-75ea-4507-84b6-c8cb13b102f9)

**Comment by sahithyandev** [comment]
Brought that down to 9.7MB. Use https://squoosh.app to optimize other images. And remove the unused ones.

![Screenshot from 2024-07-18 00-57-59](https://github.com/user-attachments/assets/b725cbcf-2753-4ab1-854a-794486da4846)

**Comment by sanjaiyan-dev** [comment]
Hi Anna,

I hope this PR will solve some of the issues by lazy loading the images. https://github.com/moraexams/website/pull/28

**Comment by sahithyandev** [comment]
No. The images are still intact. And the landing page takes 23s> to load.

**Comment by sahithyandev** [comment]
Although lazy loading is an improvement, it isn't the same as optimization of the images. The website still downloads a huge amount of images, just a bit later.

**Comment by sanjaiyan-dev** [comment]
> @sanjaiyan-dev Although lazy loading is an improvement, it isn't the same as optimization of the images. The website still downloads a huge amount of images, just a bit later.

Hmm, I think better if we compress to webp format.
